frontend/src/components/page-about/index/strategy.js

```diff
@@ -83,7 +83,7 @@ const Strategy = () => (
       </div>
     </div>
     <div>
-      <img src={StrategyImage} alt="3G's: Gather, Grow, Go" />
+      <img className="w-full" src={StrategyImage} alt="3G's: Gather, Grow, Go" />
```

Comment:
This was from when I changed the `max-w-container` to 1440px instead of 1180px. The picture would not stretch to the full width. Now that it is back to 1180px it seems that it is not needed. I can remove it.